$OpenBSD: patch-src_gmtk_media_tracker_c,v 1.1 2014/02/07 10:25:52 dcoppa Exp $

Don't force a minimum size on the tracker
(upstream svn revision r235)

--- src/gmtk_media_tracker.c.orig	Fri Feb  7 11:04:18 2014
+++ src/gmtk_media_tracker.c	Fri Feb  7 11:04:56 2014
@@ -93,7 +93,7 @@ static void gmtk_media_tracker_init(GmtkMediaTracker *
 #else
     GTK_WIDGET_UNSET_FLAGS(tracker->scale, GTK_CAN_FOCUS);
 #endif
-    gtk_widget_set_size_request(tracker->scale, 200, -1);
+    gtk_widget_set_size_request(tracker->scale, -1, -1);
     gtk_widget_show(tracker->scale);
     gtk_box_pack_start(GTK_BOX(tracker), GTK_WIDGET(tracker->scale), TRUE, TRUE, 0);
     gtk_widget_set_sensitive(tracker->scale, FALSE);
